docs/management/connectors/action-types/xmatters.asciidoc

```diff
@@ -0,0 +1,114 @@
+role="xpack"]
+[[xmatters-action-type]]
+=== xMatters connector and action
+++++
+<titleabbrev>xMatters</titleabbrev>
+++++
+
+The xMatters connector uses the https://help.xmatters.com/integrations/#cshid=Elastic[xMatters Workflow for Elastic] to send actionable alerts to on-call xMatters resources.
+
+[float]
+[[xmatters-connector-configuration]]
+==== Connector configuration
+
+xMatters connectors have the following configuration properties.
+
+Name::                                 The name of the connector. The name is used to identify a connector in the management UI connector listing, or in the connector list when configuring an action.
+Authentication Type::                                                                                                                  The type of authentication used in the request made to xMatters.
+URL:: The request URL for the Elastic Alerts trigger in xMatters. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
+Username::                                                                                                                                                      Username for HTTP Basic Authentication.
+Password::                                                                                                                                                      Password for HTTP Basic Authentication.
+
+[float]
+[[xmatters-connector-networking-configuration]]
+==== Connector networking configuration
+
+Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+[float]
+[[Preconfigured-xmatters-configuration]]
+==== Preconfigured connector type
+
+[source,text]
+--
+ my-xmatters:
+   name: preconfigured-xmatters-connector-type
+   actionTypeId: .xmatters
+   config:
+     url: https://test.host
```

Comment:
This should be updated to `configUrl` and needs `usesBasic`?

I would add two example preconfigurations, one for basic auth and one for url auth.

Reply:
@msimpson-xm Can we update the preconfigured connector example to show `configUrl` or `secretsUrl` instead of `url`?

Reply:
I'm not sure how I missed this! I will add to my latest changes :) thanks Ying